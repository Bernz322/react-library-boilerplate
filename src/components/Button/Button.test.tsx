import { render } from "@testing-library/react";
import { Button } from "..";

describe("<Button />", () => {
  it("should equal snapshot", () => {
    const { baseElement } = render(<Button label="Button"/>);
    expect(baseElement).toMatchSnapshot();
  });
});
