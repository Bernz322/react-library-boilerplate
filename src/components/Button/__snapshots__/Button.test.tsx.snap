// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should equal snapshot 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--medium storybook-button--secondary"
      type="button"
    >
      Button
    </button>
  </div>
</body>
`;
